xapian-letor/include/xapian-letor/ranker.h

```diff
@@ -256,7 +263,8 @@ class XAPIAN_VISIBILITY_DEFAULT ListNETRanker: public Ranker {
      *
      * @exception LetorInternalError will be thrown if training data is null.
      */
-    void train(const std::vector<Xapian::FeatureVector> & training_data);
+    void train(const std::vector<std::vector<Xapian::FeatureVector>>&
+	       training_data);
```

Comment:
This also doesn't need wrapping if the `Xapian::` is removed.  And another case below.